c/python/list.h

```diff
@@ -137,7 +53,12 @@ class list : public oobj {
     void set(int64_t i, oobj&& value);
     void set(int i,     oobj&& value);
 
-    friend class Arg;
+  private:
+    // Wrap an existing PyObject* into an `olist`. This constructor is private,
+    // use `py::org(src).to_pylist()` instead.
```

Comment:
Is it a typo or what is `py::org`?

Reply:
A typo. Should be `py::robj(src).to_pylist()`